eclair-core/src/main/scala/fr/acinq/eclair/channel/fsm/DualFundingHandlers.scala

```diff
@@ -109,4 +113,16 @@ trait DualFundingHandlers extends CommonFundingHandlers {
     }
   }
 
+  /**
+   * In most cases we don't need to explicitly rollback funding transactions, as the locks are automatically removed by
+   * bitcoind when transactions are published. But if we didn't publish those transactions (e.g. because our peer never
+   * sent us their signatures), their inputs may still be locked.
+   */
+  def rollbackDualFundingTxs(txs: Seq[SignedSharedTransaction]): Unit = {
```

Comment:
If our peer didn't send us their sig, shouldn't the signature be:
```suggestion
  def rollbackDualFundingTxs(txs: Seq[PartiallySignedSharedTransaction]): Unit = {
```

Reply:
We could, but it's more code to filter those out and it doesn't hurt to call `rollback` with all txs, so I opted for simplicity here.

Reply:
Actually it's better to use them all, because there is also the case where we received their remote sigs but failed to publish (either because our mempool is full or anything else), in which case we still want to unlock even `FullySignedSharedTransaction`s. The comment mentions one reason why we didn't publish, but it's not meant to be exhaustive. I'll enrich it though to ensure we don't change this signature in the future.

Reply:
How about keeping the signature as-is and filtering inside the method, or just adding a specific comment? Just to clear up the confusion, I thought I missed something when reading.

Reply:
Done in https://github.com/ACINQ/eclair/pull/2390/commits/764ec6d10e5da2faf10a20e9eacdb3f2b4940cae